templates/deploy_lambda_policy.json.tpl

```diff
@@ -0,0 +1,18 @@
+{
+  "Statement": [
+    {
+      "Action": [
+        "lambda:GetFunctionConfiguration",
+        "lambda:PublishVersion",
+        "lambda:UpdateFunctionCode",
+        "s3:GetObject",
+        "s3:PutObject"
+      ],
+      "Effect": "Allow",
+      "Resource": [
+        "${download_metadata_lambda_arn}"
```

Comment:
No0b question - does this need an S3 resource as well?

Reply:
Yep, it definitely does. I've added one in.

Reply:
As we're doing object functions I think we need the `/*` on the end of the bucket resource.